virtcontainers/sandbox.go

```diff
@@ -1062,6 +1068,19 @@ func (s *Sandbox) stopVM() error {
 		s.Logger().Info("Not stopping VM")
 		return nil
 	}
+	pid := s.hypervisor.pid()
+
+	defer func() {
+		for i := 0; i < waitRetry; i++ {
+			if running, _ := isProcRunning(pid); !running {
+				return
+			}
+			syscall.Kill(pid, syscall.SIGKILL)
+			time.Sleep(waitVMExitTime * time.Millisecond)
+		}
+		networkLogger().WithField("pid", pid).
+			Warnf("NOTE: hypervisor not exited after wait %ds", (waitRetry*waitVMExitTime)/(time.Second/time.Millisecond))
```

Comment:
I think this calculation is wrong `(50 * 200)÷(1 / 1000000) = 10000000000` isn't it?